Mathlib/CategoryTheory/Equivalence.lean

```diff
@@ -612,6 +695,15 @@ end Equivalence
 
 namespace Iso
 
+/-- Obtain a natural isomorphism between the functors of two equivalences from
+  an isomorphism in `C ≌ D`. -/
+@[simps]
+def asNatIso {e f : C ≌ D} (η : e ≅ f) : e.functor ≅ f.functor where
```

Comment:
This should be in the `Equivalence` namespace. (I do not think dot notation is an option here, there this is very specific to isomorphisms in the category of equivalences.)
I am not completely sure we need this definition, because this is also `(Equivalence.functorFunctor C D).mapIso η`.

Reply:
Right, let’s just remove it.